src/UCommerce.SiteCore.Installer/Steps/SitecoreWebconfigMerger.cs

```diff
@@ -20,7 +20,7 @@ public SitecoreWebconfigMerger(DirectoryInfo baseDirectory, DirectoryInfo siteco
 
         public async Task Run()
         {
-            _loggingService.Information<SitecoreWebconfigMerger>("Merging sitecore and ucommerce config files");
+            _loggingService.Information<SitecoreWebconfigMerger>("Starting to merge sitecore and ucommerce config files...");

```

Comment:
I think `Merging` is better then `Starting to merge`
Sitecore and Ucommerce should be capitalized

👍 to `...`